Ep2025 thank you (#1479)
EuroPython/website: src/content/pages/thank-you.mdx

- Cristián Maureira-Fredes
- Jodie Burchell
- Diego Russo
- Marina Moro
- Naa Ashiorkor Nortey
- Rodrigo Girão Serrão
- Yuliia Barabash

Big thanks to **Cristián** and **Jodie** for their outstanding efforts in leading the programme team and for making sure that speakers and participants have a smooth and enjoyable experience.

### Reviewers

Once proposals are submitted, reviewers read, rate, and leave feedback. This helps us cover many areas and ensures high-quality selections by incorporating input from multiple people.

Reviewers:

- xxx

## Speaker Mentorship
In parallel with the Call for Proposals, we kick off the Speaker Mentorship Programme. It's designed to help new and less experienced speakers submit proposals and, if accepted, build their confidence and prepare their talks.

This team coordinated the Speaker Mentorship Programme:

- Naa Ashiorkor Nortey
- Joana Owusu-Appiah
- Theofanis Petkos

### Mentors

Mentorship programme organizers matche mentees with experienced mentors who provide feedback and regular meetings.

Here are all the mentors who lent their time and expertise this year:

- xxx

## Communications & Design

> The Communications & Design team works year-round, with significant efforts during the conference launch, and major deadlines - like CFP, tickets, etc. <tobereworded>

Author: ```suggestion
The Communications & Design team works year-round, with significant efforts during the conference launch, and major deadlines - like CFP, tickets, etc. 

**needs better wording**

```

---
title: Thank You
subtitle: For the EuroPython 2025 Team
---

# Thank You

EuroPython has been organised by volunteers since 2002, driven by the passion and dedication of the community.

> What does it take to organize a conference? Let's go through it chronologically and tell you a bit more about the efforts and people behind it.

Reviewer: ```suggestion
What does it take to organise a conference? Let's go through it chronologically and tell you a bit more about the efforts and people behind it.
```

## EuroPython Society Board

Before the conference preparations even start, a board is elected annually. The board manages the day-to-day operations of the EPS, the non-profit behind the series. They're responsible for fiscal and legal work, as well as making the conference happen. That includes picking a location, finding a suitable venue, local lawyers and accountants, and finally building teams.

The board finds leads and build teams of volunteers to work in different areas.

The board members for the 2025 conference are:

- Anders Hammarquist
- Aris Nivorils
- Artur Czepiel (Chair)
- Cyril Bitterich
- Ege Akman
- Mia Bajić (Vice Chair)
- Shekhar Koirala

## Event Management - Anežka Müller

> <some intro to Anezka goes here>

Author: ```suggestion
**some intro to Anezka goes here**
```

Anežka handled communication with all suppliers, relieving volunteers and board members of logistical tasks. She coordinated with the venue, catering, speakers' dinner, social events, booth builders, t-shirts, stickers, and numerous other smaller suppliers.

> Without her support, we wouldn't have had the freedom to focus our energy on creative tasks and organizing fun events.

Reviewer: ```suggestion
Without her support, we wouldn't have had the freedom to focus our energy on creative tasks and organising fun events.
```

We are deeply grateful to Anežka for her invaluable work and dedication.

## Programme

> Right after the location is announced, the Programme Team begins work immediately. They manage Call for Proposals, Community Voting, Panel Review, final selection of the talks, tutorials and Posters,Scheduling, Keynotes and all the last minute updates.

Reviewer 2: ```suggestion
Right after the location is announced, the Programme Team begins work immediately. They manage Call for Proposals, Community Voting, Proposals Screening, Panels, final selection of the Talks, Tutorials, Posters, Scheduling, Keynotes, Open Spaces, and all the last minute updates.
```

- Cristián Maureira-Fredes

> - Jodie Burchell

Reviewer 2: ```suggestion
- Jodie Burchell
- Cristián Maureira-Fredes
```

- Diego Russo
- Marina Moro
- Naa Ashiorkor Nortey
- Rodrigo Girão Serrão
- Yuliia Barabash

> Big thanks to **Cristián** and **Jodie** for their outstanding efforts in leading the programme team and for making sure that speakers and participants have a smooth and enjoyable experience.

Reviewer 2: ```suggestion
Big thanks to **Jodie** and **Cristián** for their outstanding efforts in leading the programme team and for making sure that speakers and participants have a smooth and enjoyable experience.
```

### Reviewers

Once proposals are submitted, reviewers read, rate, and leave feedback. This helps us cover many areas and ensures high-quality selections by incorporating input from multiple people.

Reviewers:

> - xxx

Reviewer 2: ```suggestion
- Alexander Ptakhin
- Arafat Olayiwola
- Arian Ivec
- Aris Nivorlis
- Artur Czepiel
- Ashish Gupta
- BELISA RENATA ARNHOLD CENTENARO
- Bhushan Kapkar
- Celia López
- Christos Paisios
- Cosima Meyer
- Cyril Bitterich
- David Gomes
- David Louis Hollembaek
- Deborah Foroni
- Dessy
- Diego Russo
- Doreen Peace Nangira Wanyama
- Dr. Lisa Andreevna Chalaguine
- Ezra Yendau
- Goetz Buerkle
- Goutam Tiwari
- Guillermo Navas-Palencia
- Harald Nezbeda
- Irina Vidal Migallón
- Isaac Chung
- Jakub Krajniak
- Joao Futi Muanda
- Jodie Burchell
- Jose Manuel
- Joy Christine
- Juan Álvarez
- Julien Lenormand
- Kalyan Prasad
- Katerina Vankova
- Kenni Konate
- Kir Chou
- Koti Vellanki
- Krishnasis Mandal
- Kshitijaa Jaglan
- Lauris Jullien
- Liberta Gani
- Luca Corbucci
- Lucie Anglade
- MD. TAUFIQUL HAQUE KHAN TUSAR
- Marcin Gastol
- Maria Jose Molina Contreras
- Marina Moro López
- Martin Borus
- Mohit Bansal
- Nathan Bransby
- Neeraj Pandey
- Ngazetungue Muheue
- Niklas Mertsch
- Nitin Agarwal
- Nosa
- Oladapo Kayode Abiodun
- Olena
- Oliver Thieke
- Parul Gupta
- Paul Zuradzki
- Pavel Bityukov
- Roberto Polli
- Robson Kanhalelo
- Roland Sztaho
- Satish Mane
- Saurav Jain
- Sayantika Banik
- Sebastiaan Zeeff
- Shaghayegh Bagherian
- Shagun Sodhani
- Shyamnath Premnadh
- Sourav Singh
- Tathagata Dasgupta
- Tiago Montes
- Tibs
- Travis Hathaway
- Ulrik Södergren
- Varun Singhal
- Vinícius Gubiani Ferreira
- Yulia Barabash
- kt zhao
- masamori0083
- sai rahul poruri
- saucoide
- shenxianpeng
```

## Speaker Mentorship
In parallel with the Call for Proposals, we kick off the Speaker Mentorship Programme. It's designed to help new and less experienced speakers submit proposals and, if accepted, build their confidence and prepare their talks.

This team coordinated the Speaker Mentorship Programme:

- Naa Ashiorkor Nortey
- Joana Owusu-Appiah
- Theofanis Petkos

### Mentors

> Mentorship programme organizers matche mentees with experienced mentors who provide feedback and regular meetings.

Reviewer: ```suggestion
Mentorship programme organisers match mentees with experienced mentors who provide feedback and regular meetings.
```

Here are all the mentors who lent their time and expertise this year:

- xxx

## Communications & Design

The Communications & Design team works year-round, with significant efforts during the conference launch, and major deadlines - like CFP, tickets, etc.

**needs better wording**

This team is responsible for EuroPython’s social media accounts, blog, newsletter, media partnerships, and overall communications:

- Mia Bajić
- Tim Hobbs
- Martina Zátopková

The Design team creates digital and printed designs throughout the year, including banners, badges, social media cards, and website design:

- Adriana
- Cristian
- Daksh
- Mia Bajic

> - Raquel

Reviewer 3: ```suggestion
- Adriana Sánchez
- Cristián Maureira-Fredes
- Daksh P. Jain
- Mia Bajić
- Raquel Dou
```

We also maintain a conference website to ensure it includes the latest updates and essential information. This is the team responsible for keeping it updated:

> - Mia Bajic

Reviewer 3: ```suggestion
- Mia Bajić
```

- Marcin Wierzbanowski
- Artur Czepiel
- Ege Akman

Big shoutout to **Daksh** for the website design, and to **Marcin Wierzbanowski** for rewriting the entire website and refactoring our codebase.
Another big thanks to **Marcin Wierzbanowski** for helping automating design assets creation.

Thanks to **Panagiotis Kyrillos** for the EP2025 logo, based on **Adriana Sanchéz**’s astronomical clock design from 2024.

## Sponsorship

Conference income comes from ticket sales and sponsorships. The Sponsors Team contacts sponsors early, manages contracts, invoices, logistics, exhibits, and ensures all contract deliverables are fulfilled by other teams.

> The team:

Reviewer 3: ```suggestion
The team:
- Anežka Müller
```

- Artur Czepiel
- Iryna Kondrashchenko
- Jake Balaš
- Konstantin Ignatov
- Kshitijaa Jaglan
- Marcin Wierzbanowski
- Mia Bajić
- Oleh Kostromin
- Raquel Dou

## Financial Aid

Speakers or attendees who need financial assistance to attend can apply for support. This team manages financial aid applications, selects recipients, handles communications, and ensures invoices are submitted:

- Vicky Twomey-Lee
- Doreen Peace Nangira Wanyama
- Koti Vellanki
- Shekhar Koirala
- Artur Czepiel

## Operations
This is the largest team, divided into several smaller teams. You can meet them onsite or when contacting our helpdesk, staffed weekly by rotating team members.

Operations Team:

- Aneta Popelová
- Anežka Müller
- Aris Nivorlis
- George Margaritis
- George Zisopoulos
- Jakub Červinka
- Jake Balaš
- John Robert
- Kateřina Vaňková
- Liberta Gani
- Mahe Iram Khan
- Martin Borus
- Moisés Guimarães
- Niklas Mertsch
- Piotr Gnus
- Saurav Pandey
- Simon Peter Dagbui

## Discord Team

> If you have a valid ticket, you have received access to our Discord server where you can chat with fellow participants, speakers, or organizers. This team manages the Discord server, ensuring only participants have access and proper permissions.

Reviewer: ```suggestion
If you have a valid ticket, you have received access to our Discord server where you can chat with fellow participants, speakers, or organisers. This team manages the Discord server, ensuring only participants have access and proper permissions.
```

Huge thanks to **Niklas Mertsch** for leading Discord and bot management!

> The work on maintaining and updating the discord bot can be found on the [EuroPython discord bot repo](https://github.com/EuroPython/discord).

Reviewer: ```suggestion
The work on maintaining and updating the Discord bot can be found on the [EuroPython Discord bot repo](https://github.com/EuroPython/discord).
```

## A/V Team
Although an external supplier handles streaming and recording, this team communicates with the venue, manages digital screens, and handles unforeseen technical issues.

> Big thanks to **Raquel Dou** and **Piotr Gnus** for A/V coordination.

Reviewer 3: ```suggestion
Big thanks to **Raquel Dou**,  **Piotr Gnus** and **Anežka Müller** for A/V coordination.
```

## Onsite Volunteers Team

Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials. 
The onsite volunteers sub-team manages volunteer recruitment and scheduling:
- xxx

# Onsite & Remote Volunteers
Volunteers visible at the conference in yellow shirts, and those supporting remotely:

## Onsite Volunteers
- xxx 

> 

Reviewer 2: Special thanks to the volunteers that made conference a suitable place for a couple of visual impaired participants/speakers during the whole event:

- Raquel
- Konstantin
- Daniel Clauser
- Vega Čerkassova
- Aleksandr Salynskii
- Evgeniya Volkova

Reviewer 3: Remember to ask the most fantastic Jake! <3

Reviewer 3: I think we should list the broad accessibility team (mostly with Alan and Konstantin) and then shoutout to Jake, and other folks who helped with the visual impaired participants. Another shoutout to Niklas who contacted others with accessibility needs and came up with solutions to meet such needs.

## Remote Volunteers
- xxx

# Events & Workshops Organisers
Beyond talks and tutorials, EuroPython includes engaging events and workshops.

## Django Girls
A workshop designed to inspire less represented groups, particularly women, to get interested in technology. Big thanks to Doreen Peace Nangira Wanyama for organizing!

## Humble Data

> Data workshop designed for total beginners coming from less represented groups. Organizers and mentors:

Reviewer: ```suggestion
Data workshop designed for total beginners coming from less represented groups. Organisers and mentors:
```

- Cheuk Ting Ho
- Jodie Burchell
- Martin Borus
- Sena Şahin

Special thanks to **Jodie Burchell** for coordinating beginner workshops.

## Unconference Day

> In parallel with sprints and workshops, there was an unconference day featuring Q&A, discussions, and unconference-style sessions, inspired by the same concept hosted at PyCon CZ 23. Huge thanks to organizers **Daria Grudzien Linhart** and **Honza Javorek**.

Reviewer: ```suggestion
In parallel with sprints and workshops, there was an unconference day featuring Q&A, discussions, and unconference-style sessions, inspired by the same concept hosted at PyCon CZ 23. Huge thanks to organisers **Daria Grudzien Linhart** and **Honza Javorek**.
```

## PyLadies Events
On Thursday, PyLadies hosted multiple events for PyLadies and allies to get together. Shoutout to **Maria-José Molina** for coordination.

> 

Reviewer 2: ```suggestion
## Community booths

During the three main conference days, a few communities participated with a booth at the venue to tell people more about their communities and conferences. Thank you to **Maria José Molina** for coordination, and the volunteers that helped.
```

> On Thursday, PyLadies hosted multiple events for PyLadies and allies to get together. Shoutout to **Maria-José Molina** for coordination.

Reviewer 2: ```suggestion
On Thursday, PyLadies hosted multiple events for PyLadies and allies to get together. Shoutout to **Maria José Molina** for coordination.
```

> 

Reviewer 2: ## Community booths

During the three main conference days, a few communities participated with a booth at the venue to tell people more about their communities and conferences. Thank you to **Maria José Molina** for coordination, and the volunteers that helped.

## Sprints

> If you came to sprints, you could have seen open-source maintainers and people submitting PRs to projects. Many thanks to **Rodrigo Girão Serrão** for organizing sprints and supporting all maintainers-spring organizers.

Reviewer: ```suggestion
If you came to sprints, you could have seen open-source maintainers and people submitting PRs to projects. Many thanks to **Rodrigo Girão Serrão** for organising sprints and supporting all maintainers and sprint organisers.
```

# Code of Conduct Committee

We have a code of conduct in place to make sure that the conference is a safe place for many different people. This team is responsible for receiving and handling CoC reports:

> - Cheuk Ting Ho

Reviewer 3: ```suggestion
- Cheuk Ting Ho
- Georgi Ker
```

- Jakub Vysoky
- Lumír Balhar
- Vicky Twomey-Lee

----

# Acknowledgement

Apart from volunteers, we also have a few trusted suppliers we work with.

## Adriana Sánchez - Design

Huge thanks to **Adriana Sánchez** for her excellent work on digital and printed design materials!

## Gonzo - AV Team

> Big thanks to our AV team Gonzo Design who have been streaming, recording & editing our talks, and more importantly, staying with us since 2017! Here are the folks who helped us with streaming and the smooth hybrid experience:

Reviewer: Maybe link to the Gonzo website?

Reviewer 3: It's https://www.gonzodesign.it/ if you want to put it up.

- Alessandro Arese Visconti
- Ambra Lorenzetti
- Consuelo Cipriani

> - Laura Pantaleo

Reviewer 3: ```suggestion
- Claudio Cirri
- Consuelo Cipriani
- Gabriele Cardini
- Gian Marco Vigni
```

- Matteo Piccinini
- Matteo Vanni
- Raquel Dou
- Simone Giorgi
- Vincenzo Cicanese

## Conference Photographer

> The official EuroPython conference and social event photos (to be released) were taken by:

Reviewer: Maybe leave this bit out, so we don't (forget to) update it later? 
```suggestion
The official EuroPython conference and social event photos were taken by:
```

- [Jan Kaderka](https://www.jankaderka.cz/)

> EuroPython Speakers' Dinner, Social Event and Sprints photographs were taken by our volunteers

Reviewer: ```suggestion
EuroPython Speakers' Dinner, Social Event and Sprints photographs were taken by our volunteers:
```

- Mia Bajić
- Moisés Guimarães
- Kushal Das

## Susie's Babysitting

> Big thanks to [Susie's babysitting](https://www.susies.cz/en/). for making our conference more accessible!

Reviewer: ```suggestion
Big thanks to [Susie's Babysitting](https://www.susies.cz/en/). for making our conference more accessible!
```

EuroPython has been organised by volunteers since 2002, driven by the passion and dedication of the community.

What does it take to organise a conference? Let's go through it chronologically and tell you a bit more about the efforts and people behind it.

## EuroPython Society Board

Before the conference preparations even start, a board is elected annually. The board manages the day-to-day operations of the EPS, the non-profit behind the series. They're responsible for fiscal and legal work, as well as making the conference happen. That includes picking a location, finding a suitable venue, local lawyers and accountants, and finally building teams.

The board finds leads and build teams of volunteers to work in different areas.

The board members for the 2025 conference are:

- Anders Hammarquist
- Aris Nivorils
- Artur Czepiel (Chair)
- Cyril Bitterich
- Ege Akman
- Mia Bajić (Vice Chair)
- Shekhar Koirala

## Event Management - Anežka Müller

Anežka is a freelance event manager and a longtime member of the Czech Python community. This year, for the first time, we hired a contractor, Anežka, to provide paid support and improve the sustainability of our organisation.

> Anežka handled communication with all suppliers, relieving volunteers and board members of logistical tasks. She coordinated with the venue, catering, speakers' dinner, social events, booth builders, t-shirts, stickers, and numerous other smaller suppliers.

Author: ```suggestion
Anežka handled communication with all suppliers, relieving volunteers and board members of logistical tasks. She coordinated with the venue, catering, speakers' dinner, social events, sprints venue, booth builders, t-shirts, stickers, and numerous other smaller suppliers.
```

## Onsite Volunteers Team

Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials.

- xxx

# Events & Workshops Organisers
Beyond talks and tutorials, EuroPython includes engaging events and workshops.

## Django Girls
A workshop designed to inspire less represented groups, particularly women, to get interested in technology. Big thanks to Doreen Peace Nangira Wanyama for organizing!

## Humble Data

Data workshop designed for total beginners coming from less represented groups. Organisers and mentors:

- Cheuk Ting Ho
- Jodie Burchell
- Martin Borus
- Sena Şahin

Special thanks to **Jodie Burchell** for coordinating beginner workshops.

## Unconference Day
In parallel with sprints and workshops, there was an unconference day featuring Q&A, discussions, and unconference-style sessions, inspired by the same concept hosted at PyCon CZ 23. Huge thanks to organisers **Daria Grudzien Linhart** and **Honza Javorek**.

## PyLadies Events
On Thursday, PyLadies hosted multiple events for PyLadies and allies to get together. Shoutout to **Maria José Molina** for coordination.

> ## Community booths

Reviewer 3: There was also the community organiser [open space](https://ep2025.europython.eu/community-activities/#community-organisers-open-space). I know that @mjmolina, @thepetk and @cmaureir were involved (and they would know who else helped). I wonder if it might be nice to have one heading for "Community events", then "Community Organisers Open Space" and "Community Booths" as its subheadings.

Author: suggestion added, check it out https://github.com/EuroPython/website/pull/1479#discussion_r2277031716

During the three main conference days, a few communities participated with a booth at the venue to tell people more about their communities and conferences. Thank you to **Maria José Molina** for coordination, and the volunteers that helped.
## Sprints

> If you came to sprints, you could have seen open-source maintainers and people submitting PRs to projects. Many thanks to **Rodrigo Girão Serrão** for organising sprints and supporting all maintainers and sprint organisers.

Reviewer 4: Feels unfair to get this shoutout when Anežka did everything. I just smiled and waved.

# Code of Conduct Committee

We have a code of conduct in place to make sure that the conference is a safe place for many different people. This team is responsible for receiving and handling CoC reports:

- Cheuk Ting Ho
- Georgi Ker
- Jakub Vysoky
- Lumír Balhar
- Vicky Twomey-Lee

> 

Reviewer 5: ```suggestion

# Childcare

Big thanks to **Boris Kolda ** for the pre-event coordination between parents and our Childcare vendor, and the onsite support for the nannies.

```

- George Margaritis
- George Zisopoulos
- Jakub Červinka
- Jake Balaš
- John Robert
- Kateřina Vaňková
- Liberta Gani
- Mahe Iram Khan
- Martin Borus
- Moisés Guimarães
- Niklas Mertsch
- Piotr Gnus
- Saurav Pandey
- Simon Peter Dagbui

## Discord Team

If you have a valid ticket, you have received access to our Discord server where you can chat with fellow participants, speakers, or organisers. This team manages the Discord server, ensuring only participants have access and proper permissions.

Huge thanks to **Niklas Mertsch** for leading Discord and bot management!

The work on maintaining and updating the Discord bot can be found on the [EuroPython Discord bot repo](https://github.com/EuroPython/discord).

## A/V Team
Although an external supplier handles streaming and recording, this team communicates with the venue, manages digital screens, and handles unforeseen technical issues.

Big thanks to **Raquel Dou**,  **Piotr Gnus** and **Anežka Müller** for A/V coordination.

## Onsite Volunteers Team

Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials.

- xxx

> 

Author: ## Accessibility

The Accessibility team ensured that the conference, both online and onsite, was welcoming and accessible to people with diverse needs.  

Members:  
- Alan Dublaouch  
- Konstantin Ignatov  

Special thanks to the volunteers who supported our visually impaired speakers throughout the event, helping to make the conference an inclusive and comfortable space for them:  

- Jake Balaš 
- Cristián Maureira-Fredes  
- Raquel Dou  
- Konstantin Ignatov  
- Daniel Clauser  
- Vega Čerkassova  
- Aleksandr Salynskii  
- Evgeniya Volkova

> 

Author: ```suggestion
## Accessibility

The Accessibility team ensured that the conference, both online and onsite, was welcoming and accessible to people with diverse needs.  

Members:  
- Alan Dublaouch  
- Konstantin Ignatov  

Special thanks to the volunteers who supported our visually impaired speakers throughout the event, helping to make the conference an inclusive and comfortable space for them:  

- Jake Balaš 
- Cristián Maureira-Fredes  
- Raquel Dou  
- Konstantin Ignatov  
- Daniel Clauser  
- Vega Čerkassova  
- Aleksandr Salynskii  
- Evgeniya Volkova  

```

Author: @hypha, @thepetk suggestion added, see above and lmk if this is okay or if you have any more suggestions

Reviewer 3: Before the "special thanks to volunteers" paragraph, I think we should add something like this
`Big shoutout to Niklas Mertsch, who worked with most participants who indicated accessibility needs to create a support plan that suited their individual circumstances.`

# Events & Workshops Organisers
Beyond talks and tutorials, EuroPython includes engaging events and workshops.

## Django Girls
A workshop designed to inspire less represented groups, particularly women, to get interested in technology. Big thanks to Doreen Peace Nangira Wanyama for organizing!

## Humble Data

Data workshop designed for total beginners coming from less represented groups. Organisers and mentors:

- Cheuk Ting Ho
- Jodie Burchell
- Martin Borus
- Sena Şahin

Special thanks to **Jodie Burchell** for coordinating beginner workshops.

## Unconference Day
In parallel with sprints and workshops, there was an unconference day featuring Q&A, discussions, and unconference-style sessions, inspired by the same concept hosted at PyCon CZ 23. Huge thanks to organisers **Daria Grudzien Linhart** and **Honza Javorek**.

## PyLadies Events
On Thursday, PyLadies hosted multiple events for PyLadies and allies to get together. Shoutout to **Maria José Molina** for coordination.

> ## Community booths

Author: ```suggestion
## Community Events
## Community Organisers Open Space
Special thanks to **Maria José Molina**, **Theofanis Petkos**, and **Cristián Maureira-Fredes** for organising and moderating the community organisers’ space.

### Community Booths
```

Reviewer 6: ```suggestion
## Community Events

## Community Organisers Open Space

Special thanks to **Maria José Molina**, **Theofanis Petkos**, **Cristián Maureira-Fredes** and **Vassiliki Dalakiari** for organising and moderating the community organisers’ space.

```

Vassiliki helped a lot to organize the event, she was also keeping all the notes

During the three main conference days, a few communities participated with a booth at the venue to tell people more about their communities and conferences. Thank you to **Maria José Molina** for coordination, and the volunteers that helped.
## Sprints
If you came to sprints, you could have seen open-source maintainers and people submitting PRs to projects. Many thanks to **Rodrigo Girão Serrão** for organising sprints and supporting all maintainers and sprint organisers.

# Code of Conduct Committee

We have a code of conduct in place to make sure that the conference is a safe place for many different people. This team is responsible for receiving and handling CoC reports:

- Cheuk Ting Ho
- Georgi Ker
- Jakub Vysoky
- Lumír Balhar
- Vicky Twomey-Lee

# Childcare

> Big thanks to **Boris Kolda ** for the pre-event coordination between parents and our Childcare vendor, and the onsite support for the nannies.

Reviewer 3: ```suggestion
Big thanks to **Boris Kolda** for the pre-event coordination between parents and our Childcare vendor, and the onsite support for the nannies.
```
otherwise doesn't render correctly :)

- Moisés Guimarães
- Niklas Mertsch
- Piotr Gnus
- Saurav Pandey
- Simon Peter Dagbui

## Discord Team

If you have a valid ticket, you have received access to our Discord server where you can chat with fellow participants, speakers, or organisers. This team manages the Discord server, ensuring only participants have access and proper permissions.

Huge thanks to **Niklas Mertsch** for leading Discord and bot management!

The work on maintaining and updating the Discord bot can be found on the [EuroPython Discord bot repo](https://github.com/EuroPython/discord).

## A/V Team
Although an external supplier handles streaming and recording, this team communicates with the venue, manages digital screens, and handles unforeseen technical issues.

Big thanks to **Raquel Dou**,  **Piotr Gnus** and **Anežka Müller** for A/V coordination.

## Onsite Volunteers Team

Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials.
The onsite volunteers sub-team manages volunteer recruitment and scheduling:
- xxx

# Onsite
Volunteers visible at the conference in yellow shirts:

> - xxx

Author: ```suggestion
## Onsite Volunteers Team

Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials.

- xxx
```

- Simon Peter Dagbui

## Discord Team

If you have a valid ticket, you have received access to our Discord server where you can chat with fellow participants, speakers, or organisers. This team manages the Discord server, ensuring only participants have access and proper permissions.

Huge thanks to **Niklas Mertsch** for leading Discord and bot management!

The work on maintaining and updating the Discord bot can be found on the [EuroPython Discord bot repo](https://github.com/EuroPython/discord).

## A/V Team
Although an external supplier handles streaming and recording, this team communicates with the venue, manages digital screens, and handles unforeseen technical issues.

Big thanks to **Raquel Dou**,  **Piotr Gnus** and **Anežka Müller** for A/V coordination.

## Onsite Volunteers Team

Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials.
The onsite volunteers sub-team manages volunteer recruitment and scheduling:
- xxx

# Onsite & Remote Volunteers
Volunteers visible at the conference in yellow shirts, and those supporting remotely:

## Onsite Volunteers

> - xxx

Author: ```suggestion
# Onsite
Volunteers visible at the conference in yellow shirts:

- xxx
```

## PyLadies Events
On Thursday, PyLadies hosted multiple events for PyLadies and allies to get together. Shoutout to **Maria José Molina** for coordination.
## Community booths

During the three main conference days, a few communities participated with a booth at the venue to tell people more about their communities and conferences. Thank you to **Maria José Molina** for coordination, and the volunteers that helped.
## Sprints
If you came to sprints, you could have seen open-source maintainers and people submitting PRs to projects. Many thanks to **Rodrigo Girão Serrão** for organising sprints and supporting all maintainers and sprint organisers.

# Code of Conduct Committee

We have a code of conduct in place to make sure that the conference is a safe place for many different people. This team is responsible for receiving and handling CoC reports:

- Cheuk Ting Ho
- Georgi Ker
- Jakub Vysoky
- Lumír Balhar
- Vicky Twomey-Lee

----

# Acknowledgement

Apart from volunteers, we also have a few trusted suppliers we work with.

## Adriana Sánchez - Design

Huge thanks to **Adriana Sánchez** for her excellent work on digital and printed design materials!

## Gonzo - AV Team

> Big thanks to our AV team Gonzo Design who have been streaming, recording & editing our talks, and more importantly, staying with us since 2017! Here are the folks who helped us with streaming and the smooth hybrid experience:

Author: ```suggestion
Big thanks to our AV team [Gonzo Design](https://www.gonzodesign.it/) who have been streaming, recording & editing our talks, and more importantly, staying with us since 2017! Here are the folks who helped us with streaming and the smooth hybrid experience:
```

---
title: Thank You
subtitle: For the EuroPython 2025 Team
---

# Thank You

EuroPython has been organised by volunteers since 2002, driven by the passion and dedication of the community.

What does it take to organise a conference? Let's go through it chronologically and tell you a bit more about the efforts and people behind it.

## EuroPython Society Board

Before the conference preparations even start, a board is elected annually. The board manages the day-to-day operations of the EPS, the non-profit behind the series. They're responsible for fiscal and legal work, as well as making the conference happen. That includes picking a location, finding a suitable venue, local lawyers and accountants, and finally building teams.

The board finds leads and build teams of volunteers to work in different areas.

The board members for the 2025 conference are:

- Anders Hammarquist
- Aris Nivorils
- Artur Czepiel (Chair)
- Cyril Bitterich
- Ege Akman
- Mia Bajić (Vice Chair)
- Shekhar Koirala

## Event Management - Anežka Müller

> **some intro to Anezka goes here**

Author: ```suggestion
Anežka is a freelance event manager and a longtime member of the Czech Python community. This year, for the first time, we hired a contractor, Anežka, to provide paid support and improve the sustainability of our organisation.
```

- Paul Zuradzki
- Pavel Bityukov
- Roberto Polli
- Robson Kanhalelo
- Roland Sztaho
- Satish Mane
- Saurav Jain
- Sayantika Banik
- Sebastiaan Zeeff
- Shaghayegh Bagherian
- Shagun Sodhani
- Shyamnath Premnadh
- Sourav Singh
- Tathagata Dasgupta
- Tiago Montes
- Tibs
- Travis Hathaway
- Ulrik Södergren
- Varun Singhal
- Vinícius Gubiani Ferreira
- Yulia Barabash
- kt zhao
- masamori0083
- sai rahul poruri
- saucoide
- shenxianpeng

## Speaker Mentorship
In parallel with the Call for Proposals, we kick off the Speaker Mentorship Programme. It's designed to help new and less experienced speakers submit proposals and, if accepted, build their confidence and prepare their talks.

This team coordinated the Speaker Mentorship Programme:

- Naa Ashiorkor Nortey
- Joana Owusu-Appiah
- Theofanis Petkos

### Mentors

Mentorship programme organisers match mentees with experienced mentors who provide feedback and regular meetings.

Here are all the mentors who lent their time and expertise this year:

- xxx

## Communications & Design

The Communications & Design team works year-round, with significant efforts during the conference launch, and major deadlines - like CFP, tickets, etc.

> **needs better wording**

Author: ```suggestion
```

This team is responsible for EuroPython’s social media accounts, blog, newsletter, media partnerships, and overall communications:

- Mia Bajić
- Tim Hobbs
- Martina Zátopková

The Design team creates digital and printed designs throughout the year, including banners, badges, social media cards, and website design:

- Adriana Sánchez
- Cristián Maureira-Fredes
- Daksh P. Jain
- Mia Bajić
- Raquel Dou

We also maintain a conference website to ensure it includes the latest updates and essential information. This is the team responsible for keeping it updated:

- Mia Bajić
- Marcin Wierzbanowski
- Artur Czepiel
- Ege Akman

Big shoutout to **Daksh** for the website design, and to **Marcin Wierzbanowski** for rewriting the entire website and refactoring our codebase.

> Another big thanks to **Marcin Wierzbanowski** for helping automating design assets creation.

Author: ```suggestion

Another big thanks to **Marcin Wierzbanowski** for helping automating design assets creation.
```

If you have a valid ticket, you have received access to our Discord server where you can chat with fellow participants, speakers, or organisers. This team manages the Discord server, ensuring only participants have access and proper permissions.

Huge thanks to **Niklas Mertsch** for leading Discord and bot management!

The work on maintaining and updating the Discord bot can be found on the [EuroPython Discord bot repo](https://github.com/EuroPython/discord).

## A/V Team
Although an external supplier handles streaming and recording, this team communicates with the venue, manages digital screens, and handles unforeseen technical issues.

Big thanks to **Raquel Dou**,  **Piotr Gnus** and **Anežka Müller** for A/V coordination.

## Onsite Volunteers Team

Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials.
The onsite volunteers sub-team manages volunteer recruitment and scheduling:
- xxx

# Onsite & Remote Volunteers
Volunteers visible at the conference in yellow shirts, and those supporting remotely:

## Onsite Volunteers
- xxx

## Remote Volunteers

> - xxx

Author: ```suggestion
```

- Simon Peter Dagbui

## Discord Team

If you have a valid ticket, you have received access to our Discord server where you can chat with fellow participants, speakers, or organisers. This team manages the Discord server, ensuring only participants have access and proper permissions.

Huge thanks to **Niklas Mertsch** for leading Discord and bot management!

The work on maintaining and updating the Discord bot can be found on the [EuroPython Discord bot repo](https://github.com/EuroPython/discord).

## A/V Team
Although an external supplier handles streaming and recording, this team communicates with the venue, manages digital screens, and handles unforeseen technical issues.

Big thanks to **Raquel Dou**,  **Piotr Gnus** and **Anežka Müller** for A/V coordination.

## Onsite Volunteers Team

Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials.

- xxx
## Accessibility

The Accessibility team ensured that the conference, both online and onsite, was welcoming and accessible to people with diverse needs.  

Members:  
- Alan Dublaouch  
- Konstantin Ignatov  

> 

Author: ```suggestion
Big shoutout to **Niklas Mertsch**, who worked with most participants who indicated accessibility needs to create a support plan that suited their individual circumstances.
```
